build.gradle

```diff
@@ -20,3 +20,43 @@ project.ext {
         RELEASE_KEY_PASSWORD = "*****"
     }
 }
+
+// Copies unit test reports to the 'artifacts' directory
+task copyUnitTestBuildArtifacts << { task ->
+    def hashPipe = new ByteArrayOutputStream()
+    task.project.exec {
+        commandLine = ['git', 'rev-parse', '--verify', 'HEAD']
+        standardOutput = hashPipe
+    }
+
+    def destDir = "artifacts"
+    task.project.exec {
```

Comment:
We should just have a task that's ``createBuildArtifactsDirectory`` which makes the directory and writes the hash, then have the copy tasks both ``dependOn`` that task